FirebaseAppCheck/Sources/Core/FIRAppCheck.m

```diff
@@ -309,6 +321,26 @@ - (nonnull NSString *)notificationTokenKey {
   });
 }
 
+- (FBLPromise<FIRAppCheckToken *> *)retrieveLimitedTokenForcingRefresh {
```

Comment:
nit: since we don't ever call an API that takes a `forcingRefresh:` param, I think it's reasonable to remove it from this internal API's name.

Maybe something like:

```suggestion
- (FBLPromise<FIRAppCheckToken *> *)retrieveLimitedTokenPromise {
```